Core JS(Session 2 26-02-2020)/WordCount.js

```diff
@@ -0,0 +1,11 @@
+/*Question Link - http://www.codewars.com/kata/word-count*/
+
+function countWords(sentence) {
+  sentence=sentence.trim();
+  if(sentence=="") {
+   return 0;
+   }else {
+      var wordArray=sentence.split(/\s+/);
```

Comment:
it("should work with non-whitespace (ex. breakspace) chars", function() {
    Test.assertEquals(countWords("﻿Hello﻿World "), 2);
    Test.assertEquals(countWords("Hello﻿World"), 2);
  });

Please verify these tests and update logic accordingly

Reply:
For current logic these test cases are passed when I'm executing.